wordpress-mobile/gutenberg-mobile
# Update html content through the bridge

This change adds ability to update the editor content through the bridge. This is for being able to return back to a previous revision that parent app will provide(a.k.a. History feature).

![updatecontent1](https://user-images.githubusercontent.com/5032900/49377672-d5ec2200-f71b-11e8-865d-f61d405933e7.gif)

**To Test:**

Copy following content:

```

<p><b>Hello</b> World!</p>


```
Test 1 - Update content on Visual mode

- Tap (...) on right top corner, select Update HTML
- Paste the above content
- Verify that visual editor content updated with the given html
- Tap 'Save' and check the XCode logs to verify the latest html is passed
- Switch to HTML mode and verify contents are matching

Test 2 - Update content on HTML mode

- Switch to HTML editing mode
- Tap (...) on right top corner, select Update HTML
- Paste the above content
- Verify that html editor content updated with the given html
- Tap 'Save' and check the XCode logs to verify the latest html is passed
- Switch back to Visual mode and verify contents are matching

WPiOS side can be tested also with instructions here https://github.com/wordpress-mobile/WordPress-iOS/pull/10593
